### DISCUSSION TITLE
TV4 - HLS? Hur gör jag?

### DISCUSSION THREAD

**Discussion by robbenu** [stefansundin/privatkopiera#174] [General]
Hej!

Försöker få hem denna: https://www.tv4play.se/video/b423237105b26f57a0a5

hittar detta under network, men sen? 

https://vod.streaming.a2d.tv/c5740a45-6646-49ad-841b-458bef98c14c/b8a773d0-63d1-11ed-ac83-abe2ac99ce9a_20341341.ism/hls/b8a773d0-63d1-11ed-ac83-abe2ac99ce9a_20341341-video=2500000-6.m4s

https://vod.streaming.a2d.tv/c5740a45-6646-49ad-841b-458bef98c14c/b8a773d0-63d1-11ed-ac83-abe2ac99ce9a_20341341.ism/hls/b8a773d0-63d1-11ed-ac83-abe2ac99ce9a_20341341-audio=128000-6.m4s

Försöker lära mig ffmpeg nu när TV4 har joxat om så privatkopiera inte funkar där mer. 
Nån vänlig som kan förklara? Kör Win och Chrome om det är någon hjälp.

Mvh.

**Comment by Lunarwalker60** [comment]
Du får snickra ihop de ledtrådar som ges. Sök efter .m3u8 Högerklicka på den och välj Copy / Copy link address - klistra in den i ett tomt textdokument. Högerklicka sen igen på .m3u8 och välj Open in new tab. Då laddas huvudmanifestet ner till din dator som m3u8, döp om den till Manifest.m3u8 och öppna den med anteckningar. I den finns alla alternativa strömmar Du kan välja.
Bygg kommandot såhär:
ffmpeg -i "https://vod.streaming.a2d.tv/c5740a45-6646-49ad-841b-458bef98c14c/b8a773d0-63d1-11ed-ac83-abe2ac99ce9a_20341341.ism/hls/b8a773d0-63d1-11ed-ac83-abe2ac99ce9a_20341341-video=2500000.m3u" -i "https://vod.streaming.a2d.tv/c5740a45-6646-49ad-841b-458bef98c14c/b8a773d0-63d1-11ed-ac83-abe2ac99ce9a_20341341.ism/hls/b8a773d0-63d1-11ed-ac83-abe2ac99ce9a_20341341-audio=128000.m3u8" -c copy "Programnamn.mkv" -i "https://vod.streaming.a2d.tv/c5740a45-6646-49ad-841b-458bef98c14c/b8a773d0-63d1-11ed-ac83-abe2ac99ce9a_20341341.ism/hls/b8a773d0-63d1-11ed-ac83-abe2ac99ce9a_20341341-textstream_swe=1000.m3u8" "Programnamn.srt"

Rörigt, jag vet. Jag har delat upp ett exempelkommando det så att det lättare går att se hur det skall göras.

ffmpeg 
-i "https://vod.streaming.a2d.tv/
8d51cf0f-cd46-40c2-95ee-e3aa87c7b046/
b2eff210-092a-11ee-af8b-cf20ace11e5c_20586603.ism/
b2eff210-092a-11ee-af8b-cf20ace11e5c_20586603-video=800000.m3u8" 

-i "https://vod.streaming.a2d.tv/
8d51cf0f-cd46-40c2-95ee-e3aa87c7b046/
b2eff210-092a-11ee-af8b-cf20ace11e5c_20586603.ism/
b2eff210-092a-11ee-af8b-cf20ace11e5c_20586603-audio=128000.m3u8"
 -c copy "Love Island UK - S10E03.mkv" 

-i "https://vod.streaming.a2d.tv/
8d51cf0f-cd46-40c2-95ee-e3aa87c7b046/
b2eff210-092a-11ee-af8b-cf20ace11e5c_20586603.ism/
b2eff210-092a-11ee-af8b-cf20ace11e5c_20586603-textstream_swe=1000.m3u8"
 "Love Island UK - S10E03.srt"

Allt ska skrivas på EN rad utan radbrytningar
Lycka till
